src/test/fixtures/utils.js

```diff
@@ -180,6 +180,10 @@ export function waitUntilState(
   store: Store,
   predicate: State => boolean
 ): Promise<void> {
+  if (predicate(store.getState())) {
+    return Promise.resolve();
+  }
+
```

Comment:
I don't use this change anymore in this patch but this confused me while working on the tests.

Reply:
Seems reasonable.